tasks/fetch-last-zap-results.yml

```diff
@@ -0,0 +1,16 @@
+---
+inputs:
+  - name: s3-bucket-results
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: busybox
```

Comment:
Not a problem, but for my learning purposes, curious if there was a reason you went with Busybox instead of Alpine?